src/test/rgw/rgw_multi/tests.py

```diff
@@ -439,23 +441,38 @@ def gen_bucket_name():
     num_buckets += 1
     return run_prefix + '-' + str(num_buckets)
 
+def gen_role_name():
+    global num_roles
+
+    num_roles += 1
+    return "roles" + '-' + run_prefix + '-' + str(num_roles)
+
 class ZonegroupConns:
     def __init__(self, zonegroup):
         self.zonegroup = zonegroup
         self.zones = []
         self.ro_zones = []
         self.rw_zones = []
         self.master_zone = None
+
+        self.iam_zones = []
+        self.iam_ro_zones = []
+        self.iam_rw_zones = []
+        self.iam_master_zone = None
```

Comment:
why do we need to track separate lists for iam here? can't `create_role_per_zone()` loop over the existing `rw_zones`?

Reply:
removed them